default.nix

```diff
@@ -16,8 +16,8 @@ let
     src = fetchFromGitHub {
       repo = "agda-stdlib";
       owner = "input-output-hk";
-      rev = "bdfab77b179c937856c49d72321ca26e2a27d568";
-      sha256 = "+PMZjmMK5xz+3Va7RN1ErtQghzpUlzsc9JBUoL+iasc=";
+      rev = "cb72ba52dfbd4f83d1034e352eb88550a3e1f681";
+      sha256 = "+OByLIWv+pdHvWt41hniE4oeo2DJZewRyYYmNXvCix0=";
```

Comment:
Wouldn't it be more proper to depend on `agda/agda-stdlib#v1.7.3` and just add the changes in the IOHK fork as `.../Ext.agda` files, namely:
- [Unique seq equality = bag equality](https://github.com/agda/agda-stdlib/commit/668f0bbd498cfae253605fd7132c3b9ed52cc4ac) as file `Data/List/Relation/Unary/Unique/Propositional/Properties/WithK.agda`
- [Non-normalising monoid solver](https://github.com/agda/agda-stdlib/commit/bdfab77b179c937856c49d72321ca26e2a27d568) as file `Tactic.MonoidSolver.Ext` which also re-exports the standard `Tactic.MonoidSolver` as `solve-normalised` and `solve-normalised-macro`.

Reply:
I think it depends on the viewpoint. I'd like to get these in the actual stdlib at some point and in that case it's easier to do it like this.